pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go

```diff
@@ -21,6 +21,7 @@ const (
 	configNamespace                 = "openshift-config"
 	machineSpecifiedConfigNamespace = "openshift-config-managed"
 	machineSpecifiedConfig          = "kube-cloud-config"
+	externalCloudProviderFeature    = "ExternalCloudProvider"
```

Comment:
We may want to export this so that components can blacklist it when using the `FeatureGateObserver`, is this the right place to have it if so 🤔

Reply:
Let's change this and use as is. The implementation LGTM

Reply:
@sttts @deads2k Is this an appropriate place to have this constant exported? We are going to need to use it in various places across various repos to ensure it's filtered out of kube component feature flags